redis/asyncio/cluster.py

```diff
@@ -1039,16 +1059,24 @@ async def parse_response(
     async def execute_command(self, *args: Any, **kwargs: Any) -> Any:
         # Acquire connection
         connection = self.acquire_connection()
+        keys = kwargs.pop("keys", None)
 
-        # Execute command
-        await connection.send_packed_command(connection.pack_command(*args), False)
-
-        # Read response
-        try:
-            return await self.parse_response(connection, args[0], **kwargs)
-        finally:
-            # Release connection
+        response_from_cache = await connection._get_from_local_cache(args)
+        if response_from_cache is not None:
```

Comment:
Are you also check a command against a list of blacklisted/whitelisted commands before check it against cache?

Reply:
Yes 
https://github.com/redis/redis-py/blob/b5d4d29409f83e9a0056c6ed12414bf58e1ea1c0/redis/connection.py#L637